openlibrary/templates/merge_queue/merge_queue.html

```diff
@@ -80,24 +87,22 @@ <h1>$_('Community Edit Requests')</h1>
                 </div>
             </td>
             <td id="reviewer-cell-$(r['id'])">
-              $# Allow unassigning self if status is "Pending" or "Claimed" (status codes 1 and 4, respectively)
-              $if r['reviewer'] and r['reviewer'] == username and (r['status'] in [1, 4]):
+              $if can_merge and is_open and r.get('reviewer') == username:
                 $r['reviewer'] <span class="mr-unassign" data-mrid="$r['id']">&times;</span>
-              $else:
+              $elif r.get('reviewer'):
                 $r['reviewer']
             </td>
             <td>
-              $# Status code 1 is "Pending"; 4 is "Claimed"
-              $if r['status'] in [1, 4]:
-                $if can_merge and (r['reviewer'] == None or r['reviewer'] == username):
+              $if can_merge and is_open:
+                $if (not r.get('reviewer') or r.get('reviewer') == username):
                   <a class="mr-resolve-link" data-mrid="$r['id']" href="$url" target="_blank">$_('Merge')</a>
                 $elif is_submitter:
```

Comment:
See: https://github.com/internetarchive/openlibrary/pull/6738#issuecomment-1183626394

Reply:
Yes!  Changing the next `$elif is_submitter:` to an `$if` should solve this.